File: libpdf/src/main/java/com/sclbxx/libpdf/pojo/Token.java
package com.sclbxx.libpdf.pojo;

/**
 * 服务器端的Token值
 *
 * @version 1.0
 * @Author cc
 * @Date 2020/3/18-1426
 */
public class Token {
    public Data data;
    public String error;
    public int success;

    public  class Data {
        public String imageUrl;
        public int islock;
        public String password;
        public int schoolId;
        public String schoolName;
        public String schoolUniqueId;
        public String token;
        public String userId;
        public String userName;
        public int userType;
    }
}

File: libpdf/src/main/java/com/sclbxx/libpdf/pojo/Token.kt
package com.sclbxx.libpdf.pojo

import com.google.gson.annotations.SerializedName


/**
 *
 * 获取服务器端的Token值
 * @Author cc
 * @Date 2019/10/18-17:18
 * @version 1.0
 */
data class Token(
        @SerializedName("data")
        val `data`: Data,
        @SerializedName("error")
        val error: String,
        @SerializedName("success")
        val success: Int
) {

    data class Data(
            @SerializedName("imageUrl")
            val imageUrl: String,
            @SerializedName("islock")
            val islock: Int,
            @SerializedName("password")
            val password: String,
            @SerializedName("schoolId")
            val schoolId: Int,
            @SerializedName("schoolName")
            val schoolName: String,
            @SerializedName("schoolUniqueId")
            val schoolUniqueId: String,
            @SerializedName("token")
            val token: String,
            @SerializedName("userId")
            val userId: String,
            @SerializedName("userName")
            val userName: String,
            @SerializedName("userType")
            val userType: Int,
            @SerializedName("activeTime")
            val activeTime: Int

    )
}
